pkg/signer/file/local.go

```diff
@@ -34,9 +34,8 @@ type keyData struct {
 	Salt             []byte `json:"salt,omitempty"`
 }
 
-// NewFileSystemSigner creates a new signer that stores keys securely on disk.
-// If the keys don't exist at the specified paths, it generates new ones.
-func NewFileSystemSigner(keyPath string, passphrase []byte) (signer.Signer, error) {
+// CreateFileSystemSigner creates a new key pair and saves it encrypted to disk.
+func CreateFileSystemSigner(keyPath string, passphrase []byte) (signer.Signer, error) {
```

Comment:
nit: having a single function of loading or creating would be a nice to have, but maybe we want it to fail as I do see people running servers with the wrong signer by mistake

Reply:
yup yup thats what this does. it fails if it already exists